File: Pokemon/app/src/main/java/com/example/pokemon/Entrenador.kt
package com.example.pokemon

class Entrenador(
    var nombre:String,

    var color:String,
    var nivel: Int,
   var activo:Boolean?,
  var d_caminada:Double
   //var arreglo_pokemones: ArrayList<Int>

) {
    override fun toString(): String {
        return "${this.nombre} ${this.color} | ${this.activo}  ${this.nivel}  ${this.d_caminada}"
    }



    }

File: Examen I bimestre/Backend/pokemon/api/models/Entrenador.js
/**
 * Entrenador.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

    nombre:{
      type:'string',
      required:true, // Por defecto es false
    },
    color:{
      type: 'string',

    },
    nivel:{
      type:'string',

    },
    activo:{
      type:'string',

    },

   pokemones:{
      type:'string',
    }
/*
    pokemons: { // One to Many (plural)
      collection: 'pokemon', // Referencia al modelo
      via: 'entrenador' // Nombre Foreign Key en 'Pokemon'
    }
*/
  },
};
